Assets/Scripts/UI/SwitchMenus.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace UI
+{
+    public class SwitchMenus : MonoBehaviour
+    {
+        [SerializeField] private UnityEvent pausingGame = new();
+        private bool _isPaused;
```

Comment:
```suggestion
        private bool _isPaused;
        
        [SerializeField] private UnityEvent pausingGame = new();
        
```
[Events moet onder alle variabels.](https://github.com/Team-Swamp/Project-VooDoo/wiki/Code-convention#classes) Ook een line tussen de functie.